tests/widget_test.cpp

```diff
@@ -2473,71 +2473,71 @@ TEST_CASE( "Widget alignment", "[widget]" )
 
         CHECK( bp_legend.layout( ava, sidebar_width ) ==
                "      " + line1 + "\n" +
-               "            " + line2 + "\n" +
+               "            " + line2 + "  \n" +
                "           " + line3 );
 
         bp_legend._label_align = widget_alignment::RIGHT;
         bp_legend._text_align = widget_alignment::RIGHT;
 
         CHECK( bp_legend.layout( ava, sidebar_width ) ==
                "      " + line1 + "\n" +
-               "            " + line2 + "\n" +
+               "            " + line2 + "  \n" +
                "           " + line3 );
 
         bp_legend._label_align = widget_alignment::CENTER;
         bp_legend._text_align = widget_alignment::RIGHT;
 
         CHECK( bp_legend.layout( ava, sidebar_width ) ==
                "      " + line1 + "\n" +
-               "            " + line2 + "\n" +
+               "            " + line2 + "  \n" +
                "           " + line3 );
 
         bp_legend._label_align = widget_alignment::LEFT;
         bp_legend._text_align = widget_alignment::LEFT;
 
         CHECK( bp_legend.layout( ava, sidebar_width ) ==
-               line1 + "\n" +
+               line1 + "      \n" +
                line2 + "\n" +
                line3 + "           " );
```

Comment:
There are still 6 failing tests. However, there is no difference between actual and expected (using a diff tool).

E.g., for this test:

```
  "<color_c_yellow>B</color> bitten  <color_c_pink>I</color> infected
  <color_c_magenta>%</color> broken      
  <color_c_light_gray>=</color> splinted  <color_c_white>+</color> bandaged
  <color_c_light_green>$</color> disinfected  <color_c_light_red>b</color>
  bleeding           "
  ==
  "<color_c_yellow>B</color> bitten  <color_c_pink>I</color> infected
  <color_c_magenta>%</color> broken      
  <color_c_light_gray>=</color> splinted  <color_c_white>+</color> bandaged
  <color_c_light_green>$</color> disinfected  <color_c_light_red>b</color>
  bleeding           "
```

Reply:
It is weird! When comparing `length()` of both strings, I get different results: 288 vs. 274 (actual vs. expected).

Reply:
Finally found that there was no visible difference due to how the test framework prints things. Actually, there are spaces after `bandaged`.